src/histolab/tiler.py

```diff
@@ -90,25 +92,39 @@ def locate_tiles(
             The alpha level to be applied to the rescaled slide, default to 128.
         outline: str
             The outline color for the tile annotations, default to 'red'.
+        linewidth: int
+            Thickness of line used to draw tiles
+        tiles: Iterable
+            Tiles to visualize. Optional, will be extracted if None.
 
         Returns
         -------
         PIL.Image.Image
             PIL Image of the rescaled slide with the extracted tiles outlined
         """
         img = slide.scaled_image(scale_factor)
-        img.putalpha(alpha)
+        if alpha is not None:
```

Comment:
Why alpha can be none if it has a default value? I think that alpha cannot be none and if a user passes none an error would be expected, that is fine I think

Reply:
@ernestoarbitrio Actually this is on purpose. The default value is to fade out the slide image. But some users (myself included) prefer _not_ to fade out the background image, in which case they'd pass ``alpha=None``.  This is similar to how PIL.MAX_IMAGE_PIXELS has a default numerical value, but users can opt out by passing it as ``None``. Let me know if that makes sense to you :)

Reply:
Ok, I see, but IMHO `alpha` has to be 255 that means no transparency and so the image will be without any fade i think. This way we could get rid of that `if` statement.